### DISCUSSION TITLE
Can I modify or remove + add just one keychain entry?

### DISCUSSION THREAD

**Discussion by bl4ckh0l3z** [sensepost/objection#686] [Q&A]
An iOS app I'm digging stores multiple (more than 20) entries in keychain. Is there any chance to modify just one entry or remove and add just it? Thanks in advance.

**Comment by IPMegladon** [comment]
Yes, if you set up objection using the dev install (https://github.com/sensepost/objection/wiki/Development-Environment-Installation) you can use the `ios keychain update` command Introduced by #584.

**Reply by bl4ckh0l3z** [reply]
> Yes, if you set up objection using the dev install (https://github.com/sensepost/objection/wiki/Development-Environment-Installation) you can use the `ios keychain update` command Introduced by #584. 

Thanks for sharing...I was able to setup the env and spot it...but keychain looks empty by using the dump command unfortunately. The same command works fine on release version. May I respectfully ask any insight? Thanks

**Reply by IPMegladon** [reply]
Heya, found the issue. Dumb issue that slipped in while I updated the dependencies and changes some other things. 
Should be fixed in https://github.com/sensepost/objection/pull/698
